src/pages/auth/login.tsx

```diff
@@ -43,7 +44,7 @@ const LoginPage = () => {
     }
     console.log(`email: ${email}, password: ${password}`);
 
-    if (email === testAcount.username && password === testAcount.password) {
+    if (email === testAcount.email && password === testAcount.password) {
       alert(`Login realizado com sucesso. Bem vindo ${email}!!`);
```

Comment:
Let's write everything in english at first. We'll translate it later

Reply:
I didn't see that. Sorry! commum habit xD